airbyte-integrations/connectors/source-zenloop/source_zenloop/streams.py

```diff
@@ -0,0 +1,229 @@
+#
```

Comment:
are the classes in this file used?

Reply:
I'm using this [doc](https://docs.google.com/document/d/1YcSpPmGkKWXtrTh_BNx8L2L6vcuSk_KSkPqezTjtnkA/edit#) as reference. I think that we would need both until our confidence is low-code is higher and we won't need to fall back to Python. So both should be good for now but happy to hear your thoughts, @girarda.

Reply:
it's pretty easy to revert a pull request so I'd prefer we delete them, but I'm also fine with keeping them for a bit so long as we create issue to keep track of the debt